src/LinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -260,6 +262,100 @@ void UprobesUnwindingVisitor::Visit(uint64_t event_timestamp,
   listener_->OnCallstackSample(std::move(sample));
 }
 
+void UprobesUnwindingVisitor::Visit(uint64_t event_timestamp,
```

Comment:
Again, there should be at least an TODO that this needs to be refactored.

Reply:
Done.